applications/dashboard/controllers/class.logcontroller.php

```diff
@@ -241,6 +244,47 @@ public function record($recordType, $recordID, $page = '') {
         $this->render();
     }
 
+    /**
+     * Access the log history of a specific discussion thread
+     *
+     * @param string $recordID
+     * @param string $page
+     */
+    public function discussion(string $recordID, string $page = '') {
```

Comment:
I know there's some existing bluriness between applications, but since discussions are part of Vanilla, not Dashboard, I'd really like to see this functionality in that application, if at all possible. I'm not opposed to still using the Dashboard's log controller, but we should be able to extend it with Vanilla.